app/models/package_manager/base.rb

```diff
@@ -88,7 +88,10 @@ def self.save(project)
         dbproject.save
       else
         dbproject.reformat_repository_url
-        dbproject.update_attributes(mapped_project.except(:name, :releases, :versions, :version, :dependencies, :properties))
+        attrs = mapped_project.except(:name, :releases, :versions, :version, :dependencies, :properties)
+        # avoid changing the license if it has been set directly through the admin console
+        attrs = attrs.except(:licenses) if dbproject.license_set_by_admin?
```

Comment:
Do we elsewhere save the new package license, if they change license? Seems like we should always be saving the current upstream metadata, even if we override it.

Maybe on admin we should only ever change the normalized licenses not the raw licenses?

Reply:
I could move this logic into `normalize_licenses` and hopefully that would cover it for everything? I can look into updating the `normalized_licenses` directly from admin and see if that would work.

Reply:
I've updated it to handle the ignore in `normalize_licenses` instead of in the `PackageManager`. The admin controller now sets the `normalized_licenses` directly instead of messing with the `licenses` text.